src/IO/ReadHelpers.h

```diff
@@ -1038,8 +1038,9 @@ inline void readDoubleQuoted(LocalDateTime & x, ReadBuffer & buf)
 
 
 /// CSV, for numbers, dates: quotes are optional, no special escaping rules.
+/// read_bool_as_uint8 enable read "t" and "f" as UInt8 value in Hive TEXT File.
 template <typename T>
-inline void readCSVSimple(T & x, ReadBuffer & buf)
+inline void readCSVSimple(T & x, ReadBuffer & buf, bool read_bool_as_uint8 = false)
```

Comment:
After finishing https://github.com/ClickHouse/ClickHouse/pull/31072 maybe we can remove `read_bool_as_uint8` and just use `Bool` data type

Reply:
Ok.